packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -717,4 +741,119 @@ export default class CommonmarkRenderer extends Renderer {
     let rawText = yield;
     return rawText.join("");
   }
+
+  *DataSet(): Generator<void, string, string[]> {
```

Comment:
I have mixed feelings about this... it is fine, but it feels a bit odd. Do we forsee circumstances where we would want to encode the dataset?

Reply:
We might like to have a renderer where we produce the datasets in a document as CSV files or something, so I think yeah it makes sense for them to be included at rendering-time